firmware/boron-ins-fsm/test/consoleFunctionTests.cpp

```diff
@@ -91,24 +91,27 @@ SCENARIO( "Change_IM21_Door_ID", "[change door id]" ) {
 
     GIVEN( "A default global door id" ) {
         globalDoorID.byte1 = (uint8_t)strtol("AA",NULL,16);
-        globalDoorID.byte1 = (uint8_t)strtol("AA",NULL,16);
-        globalDoorID.byte1 = (uint8_t)strtol("AA",NULL,16);
+        globalDoorID.byte2 = (uint8_t)strtol("AA",NULL,16);
+        globalDoorID.byte3 = (uint8_t)strtol("AA",NULL,16);
 
-        WHEN( "the function is called with e" ) {
+        WHEN( "The function is called with e and a valid door ID was previously set" ) {
+            im21_door_id_set("00,10,00");
             int returnVal = im21_door_id_set("e");
 
-            THEN( "should return 1" ) {
-                REQUIRE( returnVal == 1);
+            THEN( "The function should return the door ID that was previously set" ) {
```

Comment:
Looks like this test is failing on Travis.

Reply:
This is strange, because the tests all pass on my machine, I'll look into what may be causing this. Same for the test cases below